packages/tpetra/core/src/Tpetra_Distributor.hpp

```diff
@@ -1142,380 +905,9 @@ namespace Tpetra {
            size_t numPackets,
```

Comment:
Eventually (not this PR), consider deprecating ArrayRCP interfaces

Reply:
This is pretty high up my list to do, but it turns out the enable_if's make this not as easy as it seems.